pkg/cmd/auth/login/login_test.go

```diff
@@ -430,6 +436,11 @@ func Test_loginRun_Survey(t *testing.T) {
 			}
 
 			assert.Equal(t, tt.wantHosts, hostsBuf.String())
+			if tt.wantErrOut == nil {
+				assert.Equal(t, "", stderr.String())
+			} else {
+				assert.True(t, tt.wantErrOut.MatchString(stderr.String()))
```

Comment:
One small nit. I think we can use https://pkg.go.dev/github.com/stretchr/testify/assert#Regexp here which would make the code a bit more concise.

Reply:
Yes, much better. 

✅